vnet/lib/vnet/configurations/vnmgr.rb

```diff
@@ -27,5 +27,19 @@ def node(&block)
         @config[:node] = Node.new.tap {|node| node.parse_dsl(&block) if block }
       end
     end
+
+    param :datapath_broadcast_mac_vendor_address, :default => [0x02, 0xbe, 0xef]
```

Comment:
This should be datapath_network_mac_..., there is no such thing as a datapath broadcast. Also I'm looking into harmonizing dp_nw with dp_rl such that both use 'mac_address'.